airbyte/_processors/sql/snowflakecortex.py

```diff
@@ -107,9 +107,10 @@ def _get_column_list_from_table(
         This is overridden due to lack of SQLAlchemy compatibility for the
         `VECTOR` data type.
         """
+        normalized_table_name = self.get_sql_table_name(table_name)
```

Comment:
Technically, the get_sql_table_name() method is supposed to take in a stream name and output the table name. And this method should receive a table name also, which should have already been normalized. While it will work in most cases, the problem would be that if there's a table prefix, that could end up getting applied twice.

Is there a way to move this operation upstream in the flow and still get a valid fix? And I think there's another method to normalize without risking adding a new prefix.


I'm on my mobile right now but will be able to dive in a bit deeper shortly...

Reply:
good point, i will take a look tomm.

Reply:
updated to use `self.normalize(...)`